developer_setup.md

```diff
@@ -37,7 +37,7 @@
   ```
 
 * Configure PostgreSQL
-  * Required PostgreSQL version is 9.4+.
+  * Required PostgreSQL version is 9.4.\*, 9.5.\*.
```

Comment:
Being picky, but the asterisks and the period look a bit goofy. `9.4.x or 9.5.x`, or just drop the patch `9.4 or 9.5` ?